samples/port-table/README.md

```diff
@@ -0,0 +1,36 @@
+# Table Specified 2-Ports Forwarding Switch
+
+This is a simple P4 program that implements a switch with two ports: 0 and 1.
+
+All traffic received via port 0 is then sent out to port 1 without modification, and vice-versa.
+
+The logic for this is implemented via an exact match control plane table with two entries.
+
+You can find additional explination about this particular program, as well as the p4 language
+in this youtube tutorial https://www.youtube.com/watch?v=qxT7DKOIk7Q&t=3151s
+
+## Running
```

Comment:
This information is duplicated (and as far as I can tell identical) in all samples. Should we factor out the common documentation into a samples/README.md file? Generally I think it's best to avoid duplication of content, both in code as well as in documentation. The individual readme's can still contain a description of what the program does, of course, since that's different for each of them.

Reply:
Good idea! This is should be ok now.